### DISCUSSION TITLE
Configuring photoview to work with another photo gallery

### DISCUSSION THREAD

**Discussion by niladrishekhar** [photoview/photoview#278] [Ideas]
Hi,
I am already using Piwigo and have the pictures located in a local host gallery. 
Can I use photoview along with it - using the Piwigo photo folders instead of uploading the whole set all over again in photoview ?

Many thanks in advance.

**Comment by viktorstrate** [comment]
Yes, you provide Photoview with a path and it will scan for media located within it.
Take a look at the [Using with Nextcloud](https://photoview.github.io/docs/usage-nextcloud/) article, it should be much of the same process.
